### ISSUE TITLE
Trailing whitespace in GCop216 pragma warning disable/restore

### ISSUE THREAD

**Issue by sungam3r** [Geeksltd/GCop#284]
Let's say we use `GCop.All.Common 2.6.0` together with `StyleCop.Analyzers 1.0.2` and have this code
```csharp
using System;

namespace ConsoleApp4
{
    class Program
    {
        static void Main()
        {
            Console.WriteLine("Hello World!");
        }

        private static void GetFoo() // GCop216 A method named `GetFoo` is expected return a value. If it's meant to be void, then use a verb other than `Get` such as Read, Download, Sync, ... 	
        {
            Console.WriteLine("foo");
        }
    }
}
```
OK. Now prohibit rule using pragma directive
```csharp
using System;

namespace ConsoleApp4
{
    class Program
    {
        static void Main()
        {
            Console.WriteLine("Hello World!");
        }

#pragma warning disable GCop216 // A method named `{0}` is expected return a value. If it's meant to be void, then use a verb other than `Get` such as Read, Download, Sync, ... 
        private static void GetFoo()
#pragma warning restore GCop216 // A method named `{0}` is expected return a value. If it's meant to be void, then use a verb other than `Get` such as Read, Download, Sync, ... 
        {
            Console.WriteLine("foo");
        }
    }
}
```
GCop216 warning gone but a StyleCop [SA1028](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1028.md) warning appeared in its place. There are trailing spaces in the description of the rule. You have to manually erase these spaces. It is annoying.

Is it possible to remove trailing spaces from the description of this rule and from all other descriptions if they are there?

**Comment by sungam3r** [comment]
Also GCop175

**Comment by Foroughi** [comment]
@sungam3r Its done,  you won't get that issue again as I`ve trimmed all the descriptions exactly before we generate them. Also, those two rules have been fixed.

**Comment by marjanjavid** [comment]
It is fixed on GCop2.6.1
